.mvn/extensions.xml

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This removal does not disable `-T Cn`  option, right? 
Can you sneak in some rationale in the commit message?

Reply:
Correct, it only removes the experimental `smart` builder which we never used (and the version here is years out of date)